scss/_patterns_grid-list.scss

```diff
@@ -1,54 +1,35 @@
 // List Grid
-
-@mixin vf-grid-list {
+@mixin vf-p-grid-list {
   $grid-border-style: 1px dotted $color-warm-grey;
```

Comment:
@grahambancroft I think we should move this into a settings file and then set it as a ```!default``` so it can be overwritten by themes if needed.